src/kubeseal_auto/kubeseal.py

```
@@ -75,188 +155,292 @@ def create_generic_secret(self, secret_params: dict):
                 if len(docs) > 1:
-                    raise ComposerError("Only single document yaml files are supported")
-                return docs[0]
-        except FileNotFoundError:
-            click.echo("Provided file does not exists. Aborting.")
-            exit(1)
-
-    def merge(self, secret_name: str):
+                    raise SecretParsingError(
+                        f"File '{secret_name}' contains multiple YAML documents. "
+                        "Only single document files are supported."
+                    )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)